src/main/webapp/WEB-INF/tags/base.tag

```diff
@@ -0,0 +1,43 @@
+<%@tag description="Main base template for chat app"%>
+<%--
+  Copyright 2017 Google Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+--%>
+<!DOCTYPE html>
+<html>
+<head>
+  <title>EastCode Chat</title>
+  <link rel="stylesheet" href="/css/main.css">
+</head>
+<body>
+
+  <nav>
+    <a id="navTitle" href="/">&lt;EastCode/&gt;</a>
+    <a href="/conversations">Conversations</a>
```

Comment:
Forgot to add the tab for the activity feed page

- [x] add activity feed tab

Reply:
Added to template; however we still need to apply the template to the rest of our pages. Will do that in a subsequent PR to reduce the complexity of this one